app/_src/introduction/kuma-requirements.md

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
This file already exists in https://github.com/kumahq/kuma-website/blob/f06a058a0f28d2186289cbf25ef727649cb57e0f/app/_src/documentation/requirements.md#L1 so why do we need both?

Reply:
@Guaris - if you resolve conversations please add a comment

Reply:
Apologies. I'll do that.